index.html

```diff
@@ -49,5 +49,203 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+
+      <!-- fourth work section -->
+
+      <div class="card">
+        <img class="img" src="/image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+<div class="bg-white">
+    <section id="aboutme">
+      <div class="me">
+        <h3 class="aboutme">About Myself</h3>
+        <p class="p">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <h4 class="h4-connect">LET'S CONNECT</h4>
+
+        <ul class="flex item-center">
+          <li class="icon-tag">
+            <a href="#"><img src="/image/twitter.png" alt="twitter" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/linkedin.png" alt="linkedin" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/m.png" alt="m" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/git.png" alt="Github" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/angelist.png" alt="angelist" /></a>
+          </li>
+        </ul>
+        <button>Get my resume</button>
+      </div>
+      <div class="base-about-myself">
+      <h4>Languages</h4>
+      <ul>
+        <li><img src="/image/framejs.png" alt="frameJs" /></li>
+        <li><img src="/image/framehtml.png" alt="framehtml" /></li>
+        <li><img src="/image/framecss.png" alt="framecss" /></li>
+      </ul>
```

Comment:
- Kindly for the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).:grinning: :grinning: :grinning: 

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

check the example below :+1: 
```
<ul>
            <li> <a href=""><img src="images/twitter.png" alt="Twitter logo" /></a></li>
            <li><a href=""><img src="images/twitter.png" alt="Twitter logo" /></a></li>
            <li><a href=""><img src="images/git.png" alt="Git logo" /></a></li>
            <li><a href=""><img src="images/m.png" alt="M logo" /></a></li>
            <li><a href=""><img src="images/nice.png" alt="V logo" /></a></li>
</ul>